CHANGELOG.rst

```diff
@@ -5,6 +5,7 @@ CHANGELOG
 1.1.dev3
 ========
 
+* bug-fix: TensorFlow: TensorBoard launched from run_tensorboard_locally=True now correctly displays updated data from training
```

Comment:
nitpicks (for consistency with the rest of the file):
* backticks for `run_tensorboard_locally=True`
* imperative mood, i.e. "display updated data from..."